### DISCUSSION TITLE
how to get shark achivments?

### DISCUSSION THREAD

**Discussion by coreymtaylor** [community/community#20440] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.com/Schweinepriester/github-profile-achievements

**Comment by FahdSeddik** [comment]
Hey!
You would need to make at least 2 pull requests that get merged into a github repo. Check out this [link](https://github.com/github-community/community/discussions/18248) too. Hope this helps!

Pull Shark checks your PR (pull request) record and show Level 
x1 - over 2 PR
x2 - over 16 PR
x3 - over 128 PR
x4 - over 1024 PR

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @Latip176 
Simply create as many PRs as you can. There are milestones to get the title like 2-PR, 16-PR, 128-PR, 1024-PR... That means, that when you have a total of 2 PRs you will win the level 1 title, and when you have a total of 16 PRs you will have a level 2 title, and so on...
The simple way is to create the PR as usual and merge it, after reaching the milestones above. You will get the shark badge from Github. 
You can see that from my Github profile 
![image](https://user-images.githubusercontent.com/71834167/178772038-9df16b13-3ed5-4674-9829-eeeff14459e3.png)

Good luck!
